FWCore/Catalog/interface/SiteLocalConfig.h

```diff
@@ -2,18 +2,18 @@
 #define FWCore_Catalog_SiteLocalConfig_h
 
 // INCLUDES
-# include <set>
-# include <string>
-# include <vector>
-# include <netdb.h>
+#include <set>
+#include <string>
+#include <vector>
+#include <netdb.h>
 
 // PUBLIC DEFINES
 // PUBLIC CONSTANTS
 // PUBLIC TYPES
 namespace edm {
-    class ParameterSet;
-    class ActivityRegistry;
-}
+  class ParameterSet;
+  class ActivityRegistry;
+}  // namespace edm
```

Comment:
For namespaces that include many lines, this added comment is nice. But for short ones like this, it just adds clutter.

Reply:
this option is all or none